clients/js/test/externalPlugins/oracle.test.ts

```diff
@@ -1161,6 +1412,24 @@ test('it can use preconfigured owner pda oracle to deny burn', async (t) => {
     ...DEFAULT_ASSET,
     asset: asset.publicKey,
     owner,
+    oracles: [
+      {
+        type: 'Oracle',
+        resultsOffset: {
+          type: 'Anchor',
+        },
+        authority: {
+          type: 'UpdateAuthority',
+        },
+        baseAddress: MPL_CORE_ORACLE_EXAMPLE_PROGRAM_ID,
+        lifecycleChecks: {
+          burn: [CheckResult.CAN_REJECT],
+        },
+        pda: {
+          type: 'PreconfiguredOwner',
+        },
```

Comment:
Technically if they chose plain address it wouldn't be a pda.  So we can either ignore that minor use case or rename this to something like `extraAccounts`

Reply:
agree - let's do that in a separate PR